index.html

```diff
@@ -20,6 +19,6 @@
     <p>
```

Comment:
Unless you have a big chunk of content, you typically don't want to break down to a new line for content tags such as `<p>`'s. Lines 19-21 can all go on the same line. 

Otherwise, well done.